build/Dockerfile-setup

```diff
@@ -1,10 +1,12 @@
-FROM debian:stretch
+FROM debian:bullseye
 MAINTAINER gustavo.amigo@gmail.com
 
+ARG CASSANDRA_URL_BASE=https://archive.apache.org/dist/cassandra/
```

Comment:
Why did you change from `https://downloads.apache.org/cassandra/` to `https://archive.apache.org/dist/cassandra/`? 🤔

Reply:
Because the requested version wasn't available in downloads.